app.rb

```diff
@@ -230,25 +211,37 @@ def render_input(key, value, name)
 
   class Directory
     def initialize(path)
-      @path = path
+      @path = Pathname.new(path).realpath.to_s
     end
 
     def route
       "/files#{path.gsub(/\A#{PROJECT_PATH}/, "")}"
     end
 
     def files
+      return @files if @files
+
       files = Dir.glob("#{path}/*").map { |file| Tint::File.new(file) }
 
-      if path != PROJECT_PATH
+      if path != Pathname.new(PROJECT_PATH).realpath.to_s
         parent = Tint::File.new(
           ::File.expand_path("..", Dir.open(path)),
```

Comment:
I think expand_path is the same as `Pathname#realpath` -- maybe we don't need `Pathname` ?